includes/class-wc-gateway-stripe.php

```diff
@@ -1094,8 +1094,7 @@ public function capture_intent( $prepared_source, $order ) {
 			return $response;
 		}
 
-		// ToDo: Check whether chrges[] only contains a single charge.
-		return $response->charges->data[0];
+		return end( $response->charges->data );
```

Comment:
Not a change request, just curiosity:

Where did you read about this, and With the way we use PaymentIntents, is it possible to:
1. Have multiple successful charges in the list?
2. An extension of the first question, but is it possible that the successful charge would not be the last one in the list?

Reply:
> Where did you read about this

[Here](https://stripe.com/docs/payments/payment-intents#at-a-glance) *"The entire history of charges attempted by a PaymentIntent is stored in the PaymentIntent’s charges property."*

> Have multiple successful charges in the list?

It's not possible with the way we're currently using PaymentIntents, but if we change that in the future this would be a super tricky bug to track down.

> An extension of the first question, but is it possible that the successful charge would not be the last one in the list?

It's not said explicitly anywhere, but I assume the `charges` list is always ordered.